SpywareFilter/sections/tracking_servers.txt

```diff
@@ -42,6 +42,11 @@
 ||visilabs.com^$third-party
 ||visilabs.net^$third-party
 !
+! pann.nate.com - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||adtg.widerplanet.com^
```

Comment:
not reproduced

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/b77b7616a50c637661f561a74c6208eeffc7d547